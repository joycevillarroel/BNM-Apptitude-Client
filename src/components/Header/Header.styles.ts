import styled from 'styled-components'

export const HeaderContainer = styled.div`
  width: 100%;
  height: 5rem;
  display: flex;
  align-items: center;
  justify-content: center;
  background-color: ${({ theme }) => theme.primary};
`

export const HeaderContent = styled.img`
  width: 20rem;
`
