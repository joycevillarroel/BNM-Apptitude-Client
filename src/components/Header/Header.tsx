import { HeaderContainer, HeaderContent } from './Header.styles'
import BNMLogo from '../../assets/images/BNM_Logo_White.png'

export function Header() {
  return (
    <HeaderContainer>
      <HeaderContent src={BNMLogo} />
    </HeaderContainer>
  )
}
