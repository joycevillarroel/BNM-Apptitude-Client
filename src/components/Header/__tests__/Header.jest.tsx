import { render } from "@testing-library/react"
import { Header } from '../Header'

jest.mock("../../../assets/images/BNM_Logo_White.png", () => {
  return {
    default: "BNM_Logo_White.png",
  };
});

describe('Card', () => {

  test('should render', () => {
    const { getByTestId } = render(<Header />)

    const component = getByTestId('header')
    expect(component).toBeTruthy()
    expect(component).toMatchSnapshot()
  })
})
