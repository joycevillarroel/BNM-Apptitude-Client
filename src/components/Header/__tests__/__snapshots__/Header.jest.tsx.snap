// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Card should render 1`] = `
<div
  class="sc-beqWaB hiRkeX"
  data-testid="header"
>
  <img
    class="sc-gueYoa jrjuMW"
    src="BNM_Logo_White.png"
  />
</div>
`;
